standalone/README.md

```diff
@@ -40,10 +40,25 @@ If you don't have servers provisioned yet, you will need to do so first. See [pr
 
 These instructions are to be followed in the [standalone](/standalone) directory.
 
-### Install ansible
+### Install local requirements
 ```bash
-brew install ansible
+brew install ansible@2.8.3 gnu-tar
 ```
+
+### Setup remote user
+
+- You will need to setup a sudo remote user with `NOPASSWD` access on the servers.
+The scripts assume an `ubuntu` user by default, this can be configured in `ansible.cfg`.
+```
+sudo visudo
+
+# At the end of the file add
+ubuntu     ALL=(ALL) NOPASSWD:ALL
+```
+Note: AWS ec2 instances already come with an `ubuntu` sudoer.
+
+- Setup your keys on the server. Make sure you can ssh as the remote user.
```

Comment:
Consider "Make sure you can establish an SSH connection to your server as a remote user".